src/iop/demosaic.c

```diff
@@ -755,7 +755,7 @@ static void xtrans_markesteijn_interpolate(float *out, const float *const in,
           for(int col = (left - sgcol + pad_rb_g + 2) / 3 * 3 + sgcol; col < mcol - pad_rb_g; col += 3)
           {
             float(*rfx)[3] = &rgb[0][row - top][col - left];
-            int h = FCxtrans(row + yoff + 18, col + xoff + 19, NULL, xtrans);
```

Comment:
@dtorop this was the only case where the increment wasn't a multiple of 6. Is it a typo or as intended?